lib/src/rules/annotate_overrides.dart

```diff
@@ -80,15 +80,21 @@ class _Visitor extends SimpleAstVisitor<void> {
     if (classElement == null) {
       return null;
     }
-    return manager.lookupInheritance(classElement, member.name);
+
+    Uri libraryUri = classElement.library.source.uri;
+    return manager
+        .getInherited(classElement.type, new Name(libraryUri, member.name))
+        ?.element;
   }
 
   @override
   void visitCompilationUnit(CompilationUnit node) {
     LibraryElement library = node == null
         ? null
         : resolutionMap.elementDeclaredByCompilationUnit(node)?.library;
-    manager = library == null ? null : new InheritanceManager(library);
+    manager = library == null
+        ? null
+        : new InheritanceManager2(library.context.typeSystem);
```

Comment:
`LinterContext` has `typeSystem`.
Ideally it should also have `inheritanceManager` instance as well.

Reply:
Cool.  Thanks!